Collaborative blog post about why it's difficult for people to write blog posts (#303)
Quansight-Labs/quansight-labs-site: posts/2022/02/what-do-we-need-to-write.md

<!--
.. title: What do we need to write?
.. slug: what-do-we-need-to-write
.. date: 2022-02-22 14:00:00 UTC+01:00
.. author: Tania Allard, Kate Brack, Tony Fast, Rohit Goswami, Eric Kelly, Mars Lee, Dharhas Pothina, Isabela Presedo-Floyd, Dillon Roach, Melissa Weber Mendonça
.. tags: Writing
.. category:
.. link:
.. description:
.. type: text
.. previewimage:
-->

We write code. We write issues. We write documentation. We write notes to 
ourselves, messages to each other, and guidelines to unite teams across 
projects.

Day in and out our remote work and open source lives are driven by written 
communication. But blog posts are one kind of writing that eludes our regular 
practice. In our weekly show and tell we got real about "why can writing blog 
posts be so hard?" and collaborated to write a blog post about what we learned 
from the discussion.

<!-- TEASER_END -->

A quick note about language: At Quansight, we have many people who are 
multilingual and/or are not working in their first language. This is the norm 

> here. Because of this, we know that there language barriers to writing that 

Reviewer: ...there are language...

were only mentioned implicitly in the discussion, but are a daily reality 
amongst the team.

## What obstacles prevent us from publishing writing?

> Writing is not a single task. There's getting the idea, organizing it, 
editing what you have, getting feedback, publishing, and so on…

> We found ourselves split into two camps (1) those that struggle to start 

Reviewer: ...camps: (1)... (add colon)

writing and (2) those that struggle to complete and publish their writing. 

> Below are their stories:

Reviewer 2: ```suggestion
Below are their stories.
```

I think a colon is not necessary here because there's no list of items right below it.

- The hardest part of writing is finishing it (especially when writing it over 
a long period of time).
- I can't afford to spend 6 hours on one sentence (and can easily do so).
- Being responsible for what you say on a company blog can be stressful.
- If I haven't looked at my drafts in a while, I don't usually find anything 
there that I feel is worth continuing on.

### Maybe it's not so bad

Some of the authors' favorite part of these lists is that many of the 
obstacles complement each other! Just like there isn't one single kind of 
author, there isn't one single set of needs people have in order to write. Do 
we need people or do we need alone time? Do we need the force of deadlines or 
the relaxation of an open-ended timeline? One of our team members even said

> It's interesting to see how everyone is struggling in different ways.

Bring those thoughts with you while we discuss the next question…

## How do people like to write?

Sometimes writing does successfully get published! 🎉 We asked our team about 
what happens the times when writing does get completed as planned. People 
commented that writing works best for them when the following happens:

> - Knowing who you are writing is important. As practice, folks suggested 

Reviewer: ...are writing to...

writing for yourself or close allies.
- Writing about topics you like (or at least tying another topic into that).

> - I like to write as part of a organized meet up where you critique and share 

Reviewer 2: ```suggestion
- Writing as part of a organized meet up where you critique and share 
```

Starting this sentence with `I` sounds a bit odd with respect to the others.

each other's work with the intent of publishing at the end. ([Writer's workshop style](https://github.com/Quansight/writers-workshop))
- Having a good editor / someone to provide constructive critiques is so helpful.
- Co-authoring is the best!
- As a way to bring people into new projects.
- It's so helpful to be hyped and have that support system.

> - To by hyped by others in the form of "I double-doggy-dare you" or "I bet a 

Reviewer: I believe the expression is "I double-dog-dare you!" :)

dollar you can."

The "I published a blog post in the last few months" team also provided some 
writing lifestyle advice:

- Write in a cabin in the woods (warning: you may risk hiking instead of 
writing)
- Drink tea while writing.
- A legal pad and a computer go together. No computer writing alone.
- Relax while writing. You'll think better. (When there's no deadline!)

> 

Reviewer: After reading about the tyranny of the blank page, I'd like some specific advice to getting up and running when facing that tyrant.  Here is a suggestion for some text to add:

"Most people agreed that to overcome the tyranny of the blank page, it's helpful to just write.  Write anything.  It can be nonsense, gibberish, garbage, or downright silly, but just write. 

If that's not effective, here are some other ideas:
- Try to sneak up on the empty page by doing something you enjoy that is related to what you want to write.  For example, if you love to draw, try drawing something related to your topic and let it pull you into that world.
- Try open-ended questions, like 'What message do I want the reader to learn?' or 'What's the coolest thing about this topic?'
- Pretend you're having a conversation with someone.  How would you start describing the topic to them?
- (more)"

I can't remember all of the ideas, so these are mine.  I'd like to see others here too.  Feel free to reject some of mine, change the order, or whatever.

## What are we to do?

Write! Write for yourself. Write for each other. 

To do this sustainably, we need to experiment with techniques that mend the 
gap between the obstacles and all the good feelings listed above.

One pattern we discovered in this discussion is that people agree writing (and 

> publishing) takes a team. Whether people struggled at the beginning or end of 

Reviewer 3: ```suggestion
publishing) takes a team. Whether people struggled at the beginning or at the end of 
```

the process, everyone mentioned needing support at one time or another. And of 
course, working with others is better when you find a way to have fun. Creating 
a system to engage in fun, consistent collaboration may be the first step to helping all our writers make it across the finish line.

In need of encouragement? Remember the wise words of another team member:

> > I can't wait to read what you want to write.

Reviewer: If that's an actual quote, then ignore this comment, but I'd put "I can't wait to read what you write."

Author: It says it was a quote in the notes, but we can ask @tonyfast how he'd like to be (secretly) quoted!
